easybuild/easyconfigs/p/P3DFFT/P3DFFT-2.7.6-gompi-2017b.eb

```diff
@@ -0,0 +1,43 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'ConfigureMake'
+
+name = 'P3DFFT'
+version = '2.7.6'
+
+homepage = 'http://www.p3dfft.net/'
+
+description = """
+ Parallel Three-Dimensional Fast Fourier Transforms, dubbed P3DFFT, as well as
+ its extension P3DFFT++, is a library for large-scale computer simulations on
+ parallel platforms.This project was initiated at San Diego Supercomputer
+ Center (SDSC) at UC San Diego by its main author Dmitry Pekurovsky, Ph.D.
+"""
+
+toolchain = {'name': 'gompi', 'version': '2017b'}
+toolchainopts = {'usempi': True, 'pic': True, 'openmp': True}
+
+# https://github.com/sdsc/p3dfft/archive/v2.7.6.tar.gz
+source_urls = ['https://github.com/sdsc/%(name)s/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['977401a42581597f3f0a60722c427946979249fffb1b7c2b106c01999720f8c0']
+
+dependencies = [
+    ('FFTW', '3.3.7'),
+]
+
+configopts = """\
+    --enable-fftw\
+    --with-fftw=$EBROOTFFTW\
+    --enable-openmp\
+    --enable-openmpi\
```

Comment:
@JackPerdue Please make this a single line:

```
configopts = "--enable-fftw --with-fftw=$EBROOTFFTW --enable-openmp --enable-openmpi"
```

Reply:
Why the requirement?   I think you'll find the style I've used is much cleaner and provides much more informative diffs when comparing different toolchain versions.   I find the repeated use of single-line configopts that go on to list a number of options less than clear.  I like to see each option spelled out distinctly so I know what I am dealing with.

Reply:
Two reasons:

* we use the single-line style in all other easyconfigs
* this will result in running a "multi-line" configure command, which is just weird

If you want to let the configure options that are different across toolchains stand out, you can do this:

```python
configopts = "--enable-fftw --with-fftw=$EBROOTFFTW --enable-openmp "
# note: options below are specific to the toolchain being used!
# OpenMPI: --enable-openmpi; Intel compilers: --enable-intel
configopts += "--enable-openmpi"